test/models/annex_test.rb

```diff
@@ -0,0 +1,36 @@
+require 'test_helper'
+
+class PlanTest < ActiveSupport::TestCase
+  setup do
+    @annex = annexes :annex_one
+  end
+
+  test 'invalid without title' do
+    @annex.title = nil
+
+    refute @annex.valid?
+
+    assert @annex.errors.added? :title, :blank
```

Comment:
Hay un "helper" casero para eso:
```suggestion
    assert_error @annex, :title, :blank
```
https://github.com/cirope/mawidabp/blob/master/test/test_helper.rb#L69

Reply:
perfecto